test/test_vmap.py

```diff
@@ -3435,6 +3435,13 @@ def op(inp, running_mean, running_var, weight, bias, training):
 
         test(self, op, tuple(inputs), in_dims=tuple(in_dims))
 
+    def test_namedtuple_returns(self, device):
```

Comment:
nit: we're testing torch.return_types returns. Let's change the name of this to `test_torch_return_types_returns` or something